.github/workflows/docker_fluentd.yml

```
@@ -48,12 +48,17 @@ jobs:
         run: echo "::set-output name=value::$(echo ${TAG_OR_BRANCH##*/}| sed 's/fluentd-//g')"
         env:
           TAG_OR_BRANCH: ${{ steps.refortag.outputs.value }}
+      - name: Path from tag
+        id: pathfromtag
+        run: echo "::set-output name=value::$(echo ${TAG:0:5})"
```

```suggestion
        run: echo "::set-output name=value::$(echo $TAG | sed -E 's/^(v[0-9]+\.[0-9]+).*/\1/')"
```
would be more robust, it could work for versions `>1.*` too if we ever need that